x-pack/legacy/plugins/siem/public/components/ml/api/anomalies_table_data.ts

```diff
@@ -22,17 +22,18 @@ export interface Body {
 }
 
 export const anomaliesTableData = async (body: Body, signal: AbortSignal): Promise<Anomalies> => {
-  const response = await fetch(`${chrome.getBasePath()}/api/ml/results/anomalies_table_data`, {
-    method: 'POST',
-    credentials: 'same-origin',
-    body: JSON.stringify(body),
-    headers: {
-      'content-Type': 'application/json',
-      'kbn-system-api': 'true',
-      'kbn-xsrf': 'true',
-    },
-    signal,
-  });
-  await throwIfNotOk(response);
-  return response.json();
+  const response = await npStart.core.http.fetch<Anomalies>(
+    '/api/ml/results/anomalies_table_data',
+    {
+      method: 'POST',
+      credentials: 'same-origin',
+      body: JSON.stringify(body),
+      headers: { 'kbn-system-api': 'true' },
+      asResponse: true,
```

Comment:
Commenting for passersby as I had to look this one up... 🙂 

https://github.com/elastic/kibana/blob/4980387c17be00fc91aef591ddcb085945c3a18e/src/core/public/http/types.ts#L219-L223

Reply:
Yes, thanks! This allowed me to continue using our `throwIfNotOk` helper, although there's an [outstanding question](https://github.com/elastic/kibana/pull/54210#discussion_r364016088) about how we want to handle errors in general.